library/L1_Peripheral/capture.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include <cstdint>
+#include "L1_Peripheral/interrupt.hpp"
+#include "utility/status.hpp"
+#include "utility/units.hpp"
+
+namespace sjsu {
```

Comment:
run clang format on this ( `make format-code` )

Reply:
When I run 'make format-code' in the /demos/sjtwo/capture folder it says 'clang-format did not modify any files' and 'make presubmit' indicates the files are OK for the whole source tree. What should I do differently?